src/Input/InputWrapper.tsx

```diff
@@ -0,0 +1,121 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+import React from "react";
+import * as typography from "../typography";
+import * as colors from "../colors";
+const IconError = require(`../../icons/IconError.tsx`).IconError;
+const IconInfo = require(`../../icons/IconInfo.tsx`).IconInfo;
+
+interface Props {
+  /**
+   * Visible description
+   */
+  description?: React.ReactNode;
+
+  /**
+   * Defaults to standard
+   */
+  size: "small" | "standard" | "large";
+
+  /**
+   * Visible title
+   */
+  label?: React.ReactNode;
+
+  /**
+   * If this exists, it will be displayed as an error message
+   */
+  error?: React.ReactNode;
+
+  /**
+   * If this exists, it will be displayed as an info message
+   */
+  info?: React.ReactNode;
+
+  /**
+   * an icon to have at the right of the input box, a search for example
+   */
+  icon?: React.ReactNode;
+
+  /**
+   * generally a Stepper or an Input component
+   */
+  children: React.ReactNode;
+}
+
+export function InputWrapper({
+  label,
+  description,
+  size = "standard",
+  error,
+  info,
+  icon,
+  children,
+}: Props): ReturnType<React.FC> {
+  return (
+    <React.Fragment>
+      <label
+        css={{
+          paddingBottom: 8,
+          ...typography.base.base,
+          fontWeight: 600,
+        }}
+      >
+        {label}
+      </label>
+      <div
+        css={{
+          ...typography.base.base,
+          color: colors.black.base,
+        }}
+      >
+        {description}
+      </div>
+      <div
+        css={{
+          marginTop: 8,
+          alignItems: "center",
+          position: "relative",
+        }}
+      >
+        {icon && (
+          <div
+            css={{
+              position: "absolute",
+              display: "inline-flex",
+              marginLeft: 10,
+              marginTop: 10,
+            }}
+          >
+            {icon}
+          </div>
+        )}
+        {children}
+        {(error || info) && (
+          <div
+            css={{
+              alignItems: "center",
+              pointerEvents: "none",
+              marginTop: 8,
+              marginRight: 8,
+              paddingLeft: size === "small" ? 8 : 12,
+              color: error ? colors.red.base : colors.grey.base,
+              ...typography.base.small,
+            }}
+          >
+            {error ? (
+              <span>
+                <IconError css={{ marginRight: 8, width: 14, height: 14 }} />
```

Comment:
To match design, it looks like the icon should be a separate column from the text and aligned to the center of the top line of text